.github/workflows/release.yml

```
@@ -73,21 +73,27 @@ jobs:
             target: aarch64
             manylinux: manylinux_2_28
 
           # macOS Apple Silicon (M1/M2/M3)
+          # Pin to Python 3.13: macos-latest ships Python 3.14 which PyO3 0.23.x
+          # does not yet support (max supported is 3.13).
           - os: macos-latest
             target: aarch64
+            python-version: "3.13"
 
           # macOS Intel
           - os: macos-13
```

actionlint (1.7.12)

[error] label "macos-13" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2025", "windows-2025-vs2026", "windows-2022", "windows-11-arm", "ubuntu-slim", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-24.04-arm", "ubuntu-22.04", "ubuntu-22.04-arm", "macos-latest", "macos-latest-xlarge", "macos-latest-large", "macos-26-intel", "macos-26-xlarge", "macos-26-large", "macos-26", "macos-15-intel", "macos-15-xlarge", "macos-15-large", "macos-15", "macos-14-xlarge", "macos-14-large", "macos-14", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)